deploy/20-deploy-clisBNB-pendle-oracle.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/dist/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { ADDRESSES } from "../helpers/deploymentConfig";
+
+enum PendleRateKind {
+  PT_TO_ASSET = 0,
+  PT_TO_SY = 1,
+}
+
+const func: DeployFunction = async ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) => {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const oracle = await ethers.getContract("ResilientOracle");
+  const proxyOwnerAddress = network.live ? ADDRESSES[network.name].timelock : deployer;
+  const addresses = ADDRESSES[network.name];
+  const ptOracleAddress = addresses.PTOracle || (await ethers.getContract("MockPendlePtOracle")).address;
+
+  await deploy("PendleOracle-PT-clisBNB-24APR2025", {
+    contract: "PendleOracle",
+    from: deployer,
+    log: true,
+    deterministicDeployment: false,
+    args: [
+      addresses["PT-clisBNB-24APR2025_Market"] || "0x0000000000000000000000000000000000000002",
+      ptOracleAddress,
+      PendleRateKind.PT_TO_SY,
+      addresses["PT-clisBNB-24APR2025"],
+      addresses.slisBNB,
+      oracle.address,
+      900,
```

Comment:
It should be 30-minute, following the Chaos Labs recommendations ([here](https://community.venus.io/t/proposal-creating-lista-pool-for-bnb-slisbnb-asbnb-pt-slisbnb-on-venus-bnb-chain/4879/9#pricingoracle-7))